File: src/server/TypeORM/src/entities/work_team.ts
import {Entity, PrimaryGeneratedColumn, Column} from "typeorm";

@Entity()
export class WorkTeam {

    @PrimaryGeneratedColumn()
    work_team_id: string;

    @Column()
    work_team_name: string;

    @Column()
    project_namework_team: string;

    @Column()
    project_name: string;

}

File: fg_metrics/VSTS/VSTSComms/VSTSComms/EntitiesMySQL/WorkTeam.cs
using System;
using System.Collections.Generic;

namespace VSTSComms.EntitiesMySQL
{
    public partial class WorkTeam
    {
        public WorkTeam()
        {
            AgileSystem = new HashSet<AgileSystem>();
            WorkTeamMember = new HashSet<WorkTeamMember>();
        }

        public string WorkTeamId { get; set; }
        public string WorkTeamName { get; set; }
        public string ProjectId { get; set; }
        public string ProjectName { get; set; }
        public bool? Active { get; set; }

        public ICollection<AgileSystem> AgileSystem { get; set; }
        public ICollection<WorkTeamMember> WorkTeamMember { get; set; }
    }
}
